Source/WebKit/WebProcess/cocoa/VideoFullscreenManager.h

```diff
@@ -190,6 +192,7 @@ class VideoFullscreenManager : public RefCounted<VideoFullscreenManager>, privat
     HashMap<PlaybackSessionContextIdentifier, int> m_clientCounts;
     WeakPtr<WebCore::HTMLVideoElement, WebCore::WeakPtrImplWithEventTargetData> m_videoElementInPictureInPicture;
     bool m_currentlyInFullscreen { false };
+    WTF::Function<void(LayerHostingContextID)> m_setupFullscreen;
```

Comment:
Nit: I'm not wild about this name, maybe `m_setupFullscreenHandler`?

Reply:
Fixed in latest patch.

Thanks for reviewing!